client/layout/popup-search/style.scss

```diff
@@ -0,0 +1,66 @@
+.popup-search__mask {
+	width: 100%;
+	height: 190vh;
```

Comment:
Could you help me understand why `190vh`? Curious to learn :)

Reply:
vh Is the percentage of the view height, so 190vh is 190% of the current view port height. It was included to make sure the mask covers the page when scrolling as well.